supervision/detection/line_counter.py

```diff
@@ -6,6 +6,7 @@
 from supervision.detection.core import Detections
 from supervision.draw.color import Color
 from supervision.geometry.core import Point, Rect, Vector
+from dataclasses import dataclass
```

Comment:
I'd drop `dataclass` here. But add `Optional` to `from typing import Dict`

Reply:
Sure! thanks @SkalskiP